packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -114,98 +415,55 @@ const slugifyName = (name) => {
 const merge = (a, b) => deepmerge(a, b, {arrayMerge: (orignal, replacement) => replacement})
 
 /**
- * @param answers - a map of package-names to package.json values that are
- *   deep-merged with each package's original package.json data. Eg:
+ * Provided a dot notation key, and a value, return an expanded object splitting
+ * the key.
+ *
+ * @example
+ * const expandedObj = expand('parent.child.grandchild': { name: 'Preseley' })
+ * console.log(expandedObj) // {parent: { child: {grandchild: {name: 'Presley}}}}
  *
- *   {
- *      "retail-react-app": {
- *          "name": "new-project-web",
- *          "version": "0.0.1",
- *      }
- *   }
+ * @param {string} key
+ * @param {Object} value
+ * @returns
  *
- * Each package name included in the object's keys will be copied into the
- * generated project, all others are excluded.
  */
-const runGenerator = (answers, {outputDir, verbose}) => {
-    checkOutputDir(outputDir)
-    // Excluding pwa-kit-create-app, these are the public pwa-kit-* packages that can be installed through NPM.
-    const npmInstallables = ['pwa-kit-react-sdk', 'pwa-kit-dev', 'pwa-kit-runtime']
-
-    // Check specified SDK versions actually exist on NPM.
-    npmInstallables.forEach((pkgName) => {
-        const versions = JSON.parse(sh.exec(`npm view ${pkgName} versions --json`, {silent: true}))
-        if (versions.indexOf(SDK_VERSION) < 0) {
-            const msg =
-                `Error: You're generating a project using version "${SDK_VERSION}" of ` +
-                `PWA Kit, but "${pkgName}@${SDK_VERSION}" does not exist on NPM.\n` +
-                `The available versions are:\n${versions.map((v) => `  ${v}`).join('\n')}`
-            console.error(msg)
-            process.exit(1)
-        }
-    })
-
-    extractTemplate('template-retail-react-app', outputDir)
-
-    const pkgJsonPath = p.resolve(outputDir, 'package.json')
-    const pkgJSON = readJson(pkgJsonPath)
-    const pkgDataWithAnswers = merge(pkgJSON, answers['retail-react-app'])
-
-    npmInstallables.forEach((pkgName) => {
-        const keys = ['dependencies', 'devDependencies']
-        keys.forEach((key) => {
-            const deps = pkgDataWithAnswers[key]
-            if (deps && deps[pkgName]) {
-                deps[pkgName] = SDK_VERSION
-            }
-        })
-    })
-
-    writeJson(pkgJsonPath, pkgDataWithAnswers)
-
-    const manifest = p.resolve(outputDir, 'app', 'static', 'manifest.json')
-    const siteName = pkgDataWithAnswers.name
-    replaceJSON(manifest, {
-        name: siteName,
-        short_name: siteName,
-        start_url: '/?homescreen=1',
-        icons: [
-            {
-                src: './img/global/app-icon-192.png',
-                sizes: '192x192'
-            },
-            {
-                src: './img/global/app-icon-512.png',
-                sizes: '512x512'
-            }
-        ]
-    })
```

Comment:
This comes from npm now?

Reply:
Ok so that is located in the bootstrap templated manifest file now. Additionally it's in a per-template asset file for when you are creating a project that isn't using extensibility. 

Ideally in the future all out templates are generated using extensibility and there would only be one occurrence of this, but currently because we allow users to opt out of extensibility, we need to have it else where. I was originally going to reference that handlebars file from the bootstrap folder, but it didn't feel right.